app/components/how-do-you-feel.jsx

```diff
@@ -0,0 +1,95 @@
+//https://github.com/EnCiv/civil-pursuit/issues/62
+import React, { useState } from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+
+function Howdoyoufeel(props) {
+    const {
+        className = '',
+        onDone = () => { },
+        title = '',
+        disabled = false,
+        disableOnClick = false,
+        children,
+        ...otherProps
+    } = props
+    const [isDisabled, setIsDisabled] = useState(disabled)
+    const classes = buttonStyles()
+    const combinedClassName = cx(classes.buttonBase, className)
+
+    return (
+        <div>
+            <p style={{ fontSize: "35px", fontWeight: "normal" }}>How do you feel about it</p>
+            <span>
+                <input className={combinedClassName}
+                    title={title} disabled={isDisabled} type="Submit" defaultValue="Awesome!"
+                    onClick={() => { onDone() }}
```

Comment:
onDone({valid: true, value: "Awesome!"})  - the parent passes in onDone, and needs to receive the result of the users action.  kind of like the way that onClick works, but more data oriented.